src/errors/duplicateParamsError.ts

```diff
@@ -1 +1,5 @@
-export class DuplicateParamsError extends Error {}
\ No newline at end of file
+export class DuplicateParamsError extends Error {
+  public constructor(paramName: string) {
+    super(`Invalid Param "${paramName}": Router can not have duplicate param names in path and query.`)
```

Comment:
For some reason github isn't letting me make a suggestion here. But slight tweak

```
`Invalid Param "${paramName}": Router does not support multiple params by the same name. All param names must be unique.`
```